models/vyos/vyos_firewall/vyos_firewall.yaml

```diff
@@ -0,0 +1,212 @@
+---
+GENERATOR_VERSION: '1.0'
+
+ANSIBLE_METADATA: |
+    {
+        'metadata_version': '1.1',
+        'status': ['preview'],
+        'supported_by': 'network'
+    }
+
+NETWORK_OS: vyos
+RESOURCE: firewall_global
+COPYRIGHT: Copyright 2019 Red Hat
+
+DOCUMENTATION: |
+  module: vyos_firewall_global
+  version_added: 2.10
+  short_description: Manage global policies or configurations for firewall on VyOS devices..
+  description: This moduleglobal policies or configurations for firewall on VyOS devices.
+  author:
+     - Rohit Thakur (@rohitthakur2590)
+  options:
+    config:
+      description: A dictionary of Firewall global configuration options.
+      type: dict
+      suboptions:
+        afi:
+          description:
+            - Specifies IP address type.
```

Comment:
need to update description in context to list.

Reply:
@rohitthakur2590 is there any specific reason of keeping the params like source_route under afi, as IMO since afi is just address family indicator it should be accepting ipv4/v6 as input and also from internal code logic we use it just for differentiating b/w firing ipv4/v6 commands.